### PR TITLE
add source code hyperlink for HaaS EDU K1 version check description

### PR DESCRIPTION
**Pull request by ethanlcz**

### CODE REVIEW

**Comment by skylarCai** [comment]
感谢提交 Pull Requests！
Thanks for submitting a pull request.

**Comment by skylarCai** [comment]
auto build success,number:206

**Comment by skylarCai** [comment]
autotest fail,number:1546

**Review by wushengtws — APPROVED** [review]

File: haas_lib_bundles/python/libraries/haaseduk1/README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ HaaS EDU K1开发板硬件有两个版本，两个版本内置的的外围传感
 本文主要说明如何用判断HaaS EDU K1的硬件版本和固件版本。
```
Comment (skylarCai) [inline haas_lib_bundles/python/libraries/haaseduk1/README.md:18]: LGFM

**Review by skylarCai — COMMENTED** [review]
no security risk

**Pull request merged**